### PR TITLE
[Chat][Feature] Add message input component to chat screen

### PR DESCRIPTION
**Pull request by Mrayyan**
## Purpose

Added message input component in chat screen along side send message button

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Pull Request Checklist

- [ ] Public API changes
- [ ] Verified for cross-platform
- [ ] Synced with iOS, Web team
- [ ] Internal review completed
- [ ] UI Changes
  - [ ] Screen captures included
  - [ ] Tested for Light/Dark mode
  - [ ] Tested for screen rotation
  - [ ] Tested on Tablet and small screen device (5")
  - [ ] Include localization changes
- [ ] Tests
  - [ ] Memory leak analysis performed
  - [ ] Tested on API 21, 26 and latest 
  - [ ] Tested for background mode
  - [ ] Unit Tests Included
  - [ ] UI Automated Tests Included

## How to Test

* Open...
* Click on...

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: azure-communication-ui/gradle.properties [hunk 1]
```diff
@@ -20,4 +20,4 @@ android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
 # prevent build failures when ADO Pipeline builds the test APK
-android.useNewApkCreator=false
+android.useNewApkCreator=true
```
Comment (ahammer) [inline azure-communication-ui/gradle.properties:23]: Is this change necessary?

File: azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/ChatCompositeBottomBar.kt [hunk 2]
```diff
@@ -4,24 +4,32 @@
...
-                value = "",
-                onValueChange = {},
-                modifier = Modifier.fillMaxWidth(fraction = 0.8f)
+        Row(
+            horizontalArrangement = Arrangement.SpaceEvenly,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+
+            MessageInputView(
+                textColor = Color("#212121".toColorInt()),
```
Comment (ahammer) [inline azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/ChatCompositeBottomBar.kt:27]: Move these colors to the Theme, and reference them internally in the component. We don't need to pass them via the composable parameters.

Comment (Mrayyan) [reply azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/ChatCompositeBottomBar.kt]: Moved

**Review by ahammer — DISMISSED** [review]

**Review by ShaunaSong — DISMISSED** [review]

**Review by lpeixin — DISMISSED** [review]

**Review by ShaunaSong — APPROVED** [review]

**Review by ahammer — APPROVED** [review]
